test/uk/gov/hmrc/trusts/services/ValidationServiceSpec.scala

```diff
@@ -52,6 +51,13 @@ class ValidationServiceSpec extends BaseSpec {
         validator.validate[ExistingTrustCheckRequest](jsonString).isLeft mustBe true
       }
     }
-  }
+    "return a list of validaton errors " when {
+      "json request is valid but failed in business rules. " in {
+        val jsonString = JsonUtils.getJsonFromFile("trust-business-validation-fail.json")
```

Comment:
Why does this fail? Can we be specific about the case that it fails on?

Reply:
updated heading.